lib/internal/dns/utils.js

```diff
@@ -23,10 +24,20 @@ const {
   ERR_INVALID_OPT_VALUE
 } = errors.codes;
 
+function validateTimeout(options) {
+  let timeout = -1;  // Use c-ares's default timeout.
+  if (options != null && typeof options === 'object' && 'timeout' in options) {
+    timeout = options.timeout;
+    validateInt32(timeout, 'options.timeout', -1, 2 ** 31 - 1);
+  }
+  return timeout;
```

Comment:
```suggestion
  const { timeout = -1 } = { ...options };
  validateInt32(timeout, 'options.timeout', -1, 2 ** 31 - 1);
  return timeout;
```

Bit more concise.

Reply:
I also prefer this to prevent `in` being used.

Reply:
Ping @bnoordhuis